ArcGISRuntimeSDKQt_CppSamples/LocalServer/LocalServerMapImageLayer/LocalServerMapImageLayer.cpp

```diff
@@ -78,9 +77,15 @@ void LocalServerMapImageLayer::connectSignals()
   {
     if (LocalServer::status() == LocalServerStatus::Started)
     {
-      // set temp path
-      QTemporaryDir tempDir;
-      LocalServer::instance()->setTempDataPath(tempDir.path());
+      // create temp path
+      const QString tempPath = QDir::homePath() + "/EsriQtTemp";
```

Comment:
This could be a much longer path now on Linux. `/tmp` is about as short as you can hope for.